tests/Rector/v12/v0/typo3/HintNecessaryUploadedFileChangesRector/Fixture/fixture.php.inc

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace Ssch\TYPO3Rector\Tests\Rector\v12\v0\typo3\HintNecessaryUploadedFileChangesRector\Fixture;
+
+use TYPO3\CMS\Core\Resource\ResourceStorage;
+
+final class CustomResourceStorage extends ResourceStorage
+{
+    public function addUploadedFile(array $uploadedFileData)
+    {
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Ssch\TYPO3Rector\Tests\Rector\v12\v0\typo3\HintNecessaryUploadedFileChangesRector\Fixture;
+
+use TYPO3\CMS\Core\Resource\ResourceStorage;
+
+final class CustomResourceStorage extends ResourceStorage
+{
+    public function addUploadedFile(array $uploadedFileData)
```

Comment:
where is that comment?

Reply:
I added that comment to enforce the code samples to be valid.
But yes, we could think of the message to be added into the function to hint not only via CLI, but also within the method for the custom changes

Reply:
That would be great. I thought this is the case on other rectors as well but they are all the same. A `TODO: ...` comment would be great.

Reply:
will try that then.

Reply:
I would put the comment at the end of the line if possible and then check if a `TODO: ` comment exists already before adding it.

Reply:
Easy, bro!

Reply:
Ups, I just noticed that I kinda messed up your last force push. Sorry. I adjusted my code according to yours, just slightly different.